custom_components/plugwise-beta/__init__.py

```diff
@@ -148,14 +154,17 @@ def async_remove_listener(self, update_callback):
 
     async def async_refresh_all(self, _now: Optional[int] = None) -> None:
         """Time to update."""
-        _LOGGER.debug("Plugwise Smile updating with interval: %s", self.update_interval)
+        _LOGGER.debug("Smile updating with interval: %s", self.update_interval)
         if not self.listeners:
-            _LOGGER.error("Plugwise Smile has no listeners, not updating")
+            _LOGGER.error("PSmile has no listeners, not updating")
             return
 
-        _LOGGER.debug("Plugwise Smile updating data using: %s", self.update_method)
+        _LOGGER.debug("Smile updating data using: %s", self.update_method)
 
-        await self.api.full_update_device()
+        update_success, update_status = await self.api.full_update_device()
+        if not update_success:
+            _LOGGER.error("Smile update failed: %s", update_status)
```

Comment:
Raise a PlatformNotReady here as well?

Reply:
From what I found in HA-core homeassistant/helpers/entity_platform.py it's only used during setup, not on updateloops?

Reply:
Ok, understood.